autorest/version.go

```diff
@@ -18,11 +18,14 @@ var version string
 // Version returns the semantic version (see http://semver.org).
 func Version() string {
```

Comment:
Thanks @cosmincojocar! However, there are still a couple of tweaks needed here:
- The declaration, `var once sync.Once` should be made outside of the `Version` function.
- `once.Do(func(){...}` is an alternative to `if version == "" {...}`. It is safe to remove that check.